### ISSUE TITLE
Create ARM build for Debian Buster to support older GLIBC version

### ISSUE THREAD

**Issue by monpelaud** [Cuperino/QPrompt#28]
Hi,

The latest Qprompt's Debian package (qprompt-1.0.0-Linux-aarch64.deb) requires glibc 2.34 but 64-bit Raspberry Pi OS Bullseye uses glibc 2.31:

/lib/aarch64-linux-gnu/libc.so.6: version 'GLIBC_2.34' not found (required by ./qprompt)
/lib/aarch64-linux-gnu/libstdc++.so.6: version 'GLIBCXX_3.4.29' not found (required by ./qprompt)

Is it possible to use Qprompt on a 64-bit Bullseye Raspberry Pi 4?

Many thanks for your help and for providing Qprompt.

**Comment by Cuperino** [comment]
HI @monpelaud,

> Is it possible to use Qprompt on a 64-bit Bullseye Raspberry Pi 4?

Not at the time. At least not without recompiling from either Raspberry OS Bullseye or Debian Bullseye using aarch64 architecture. For now, use Ubuntu 21.10 for the Raspberry Pi to run QPrompt's 1.0 build: https://ubuntu.com/download/raspberry-pi

qprompt-1.0.0-Linux-aarch64 was created from Ubuntu 21.10 on an aarch64 VM. I've tried to get a Debian Bullseye aarch64 VM running to no avail. I will use a Raspberry Pi Bullseye VM next.

I'm no longer using a physical Raspberry Pi to compile, just to save time. QPrompt needs at least 6 GB of RAM to compile successfully, which means setting up additional swap space on the my Raspberry Pi 400. I also recently had to wipe the SD card to use it on the PinePhone Pro. Support for the PinePhone is on its way and will come the form of a Flatpak.

I believe the first beta build for aarch64 was compiled from a Raspberry Pi running Raspberry OS Bullseye. This beta should work for you out of the box: https://github.com/Cuperino/QPrompt/releases/tag/v1.0.0-beta-005
